tests/scripts/run-test-suites.pl

```diff
@@ -16,6 +16,7 @@
 # Options :
 #   -v|--verbose    - Provide a pass/fail/skip breakdown per test suite and
 #                     in total
+#   -d|--debug      - Print debug output when a test suite fails
```

Comment:
I think it needs to be clear that a failing test suite is executed twice, and that the debug output comes from the second execution.